usvm-ts/src/test/kotlin/org/usvm/samples/Arguments.kt

```diff
@@ -19,4 +18,41 @@ class Arguments : TSMethodTestRunner() {
             )
         )
     }
+
+    @Test
+    fun testSingleArg() {
+        discoverProperties<TSObject.TSNumber>(
```

Comment:
Please, add properties matchers to tests. Otherwise, for now they don't check anything but the machine produced something